src/wayland/seat/keyboard/mod.rs

```diff
@@ -544,6 +545,13 @@ impl KeyboardHandle {
         let ret = tempfile().and_then(|mut f| {
             f.write_all(self.arc.keymap.as_bytes())?;
             f.flush()?;
+
+            if kbd.version() >= 7 {
+                if let Err(err) = nix::fcntl::fcntl(f.as_raw_fd(), FcntlArg::F_GET_SEALS) {
```

Comment:
Is this call actually setting any kind of seals on the file?

Reply:
My bad, the error log is misleading. This is basically a check if selling is supported, so the log should just be "Sealing not supported".

(If selling is supported, we should then return an existing fd of a sealed file)

I guess this log does not make much sense if the whole thing is not implemented.
I guess I'll implement the whole thing, it is supported on Linux and FreeBSD and that's the only targets we care about anyway.

Reply:
Done, replaced with seal support